src/express.re

```diff
@@ -268,7 +268,7 @@ module App = {
   external asMiddleware : t => Middleware.t = "%identity";
   /** [asMiddleware app] casts an App instance to a Middleware type */
 
-  external listen : t => port::int => unit = "" [@@bs.send];
+  external listen : t => port::int => onListen::(Js.Null_undefined.t Js.Exn.t => unit) [@bs.uncurry] => unit = "" [@@bs.send];
```

Comment:
how do you feel about refactoring this a little so that port and onListen have default arguments? that way this doesn't have to be a breaking change. Something like:

```reason
external listen_ : t => port::int => onListen::(Js.Null_undefined.t Js.Exn.t => unit) [@bs.uncurry] => unit = "" [@@bs.send];

let listen app ::port=3000 onListen=(fun _ => ()) => listen_ app ::port ::onListen;
```
```